src/inspect_ai/solver/_basic_agent.py

```diff
@@ -54,6 +54,7 @@ def basic_agent(
     max_attempts: int = 1,
     message_limit: int | None = None,
     token_limit: int | None = None,
+    max_output: int | None = None,
```

Comment:
Does this want to be `max_tools_output` ?

Reply:
changed to `max_tool_output`